app/wrappers/device/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react'; // eslint-disable-line
```

Comment:
Our components library has `DeviceProvider` implementation. Don't need to create duplication.